README.md

```diff
@@ -2,7 +2,8 @@
 
 ## Purpose
 
-This is the User Admin UI for the [GovWiFi][link_govwifi] project.
+This is the admin platform, a website where organisations can create and manage their GovWifi installation within their organisation.
+The Govwifi page can be found [here][link_govwifi].
```

Comment:
Is this link useful?

Reply:
Not completely sure why it added before.
I was thinking maybe we needed to direct to the live page in some way.